### ISSUE TITLE
Add opt_in_flag schema for MHV

### ISSUE THREAD

**Issue by nsoutouras** [department-of-veterans-affairs/va.gov-team#46578]
# Description
As part of their rollout of the new MHV experience, MHV needs us to add a flag for tracking if a user has opted in to the new experience. This ticket covers the creation of a schema to track MHV opt_in_flags within vets-api.

# Acceptance Criteria
- [ ] Update `vets-api` database to include new `mhv_opt_in_flags` table
- [ ] `mhv_opt_in_flags` table columns:
  * `user_account_uuid` => UUID
  * `opt_in_flag` => string
  * `timestamp` => timestamp

**Comment by patrickvinograd** [comment]
There was an earlier frontend/backend feature called `beta_registrations` that could be appropriate to resurrect here. It had not been used for a couple years so I recently decommissioned it, but it could be pulled back in:
vets-api decommissioning PR

**Comment by patrickvinograd** [comment]
This does not solve for the `Flag status is readable/writeable for MHV` AC - but also I don't know for sure if that is required.

**Comment by bosawt** [comment]
Questions:
- Is there any need for the VA frontend to delete or re-set an opt_in_flag setting? Or is it the case that once a user is `opt_in_flag=true` that will always be the case for that user?
- When a flag is deprecated and we start to ignore it from MHV/VA frontend (when it reaches phase 3 according to the diagrams we went over recently), is there any need to audit any aspect of who opted in and who didn't, or can't we simply destroy the opt_in_flag for the specific feature when it reaches that phase?
- It seems like it makes sense to prioritize this work for VA.gov interactions, and separate this from potential MHV readable/writeable, so just want to confirm that we'll hold off on that aspect until we've determined it is necessary (and we'll probably have to work with MHV to implement, etc when that happens)
- I think we went over this, but wanted to confirm, is there any identity team frontend requirement for this work?
